neon_utils/skills/mycroft_skill.py

```diff
@@ -293,12 +302,14 @@ def _wait_response(self, is_cancel, validator, on_fail, num_retries,
                     LOG.info("No user response")
                     return None
             else:
-                if validator(response):
-                    return response
-
                 # catch user saying 'cancel'
                 if is_cancel(response):
                     return None
+                validated = validator(response)
```

Comment:
Does this need to override the method from `ovos_workshop` anymore?

Reply:
Requires at least the next ovos-workshop stable release so this should stay for now